### PR TITLE
[v13] feat: adds motd to the ui

### PR DESCRIPTION
**Pull request by flyinghermit**
Backport #27813 to branch/v13

### CODE REVIEW

**Review by ryanclark — APPROVED** [review]

**Review by kimlisa — APPROVED** [review]

**Comment by zmb3** [comment]
Noticed one small issue. We're not properly respecting whitespace and multiline MOTD.

image

Let's hold off on backports until we fix this.

**Review by zmb3 — CHANGES_REQUESTED** [review]
Let's fix multiline MOTD before merging.

**Review by zmb3 — APPROVED** [review]
We can release this and add multiline support in a follow up.

**Pull request merged**
